src/cli/cli.cc

```diff
@@ -2592,7 +2592,7 @@ int main (const int argc, const char* argv[]) {
       Path pathPackage = paths.platformSpecificOutputPath / (
         settings["build_name"] + "_" +
         settings["meta_version"] + "_" +
-        replace(platform.arch, "x86_64", "amd64")
+        replace(SSC::platform.arch, "x86_64", "amd64")
```

Comment:
just linking https://github.com/socketsupply/socket/issues/568 here, so we don't forget to fix it here later as well